resize.php

```diff
@@ -0,0 +1,71 @@
+<?php
+require(dirname(__FILE__).'/config/config.inc.php');
+$url = filter_input(INPUT_GET, 'url');
+
+preg_match('/^(img\/([^\/]*)\/(?:\d\/)*)(\d+)-([_a-z0-9]+)\.([a-z]+)$/', $url, $matches);
+$url_path = $matches[1];
+$img_path = $matches[2];
+$id_image = $matches[3];
+$size_name = $matches[4];
+$ext = $matches[5];
+switch ($img_path) {
```

Comment:
If we put this logic into a front controller, we might be able to reuse some already-in-the-codebase logic and we can not write this switch